com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryActionEditor.cs

```diff
@@ -108,6 +108,60 @@ public class VRCFuryActionDrawer : PropertyDrawer {
 
                 return row;
             }
+            case nameof(PoiyomiUVTileAction): {
+                var content = new VisualElement();
+                var row = new VisualElement {
+                    style = {
+                        alignItems = Align.FlexStart,
+                        flexDirection = FlexDirection.Row,
+                    }
+                };
+
+                var label = new Label("Poiyomi UV Tile Discard") {
+                    style = {
+                        flexGrow = 0,
+                        flexBasis = 140,
+                    }
+                };
+                row.Add(label);
+
+                var propField = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("renderer"));
+                propField.style.flexGrow = 1;
+                row.Add(propField);
+            
+                var propField2 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("slot1"), tooltip: "UV X Axis, Val 0-3");
+                propField2.style.flexGrow = 0;
+                propField2.style.flexBasis = 30;
+                row.Add(propField2);
+
+                var propField3 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("slot2"),tooltip: "UV Y Axis, Val 0-3");
+                propField3.style.flexGrow = 0;
+                propField3.style.flexBasis = 30;
+                row.Add(propField3);
+                content.Add(row);
+
+                var row2 = new VisualElement {
+                    style = {
+                        alignItems = Align.FlexStart,
+                        flexDirection = FlexDirection.Row,
+                    }
+                };
+
+                var adv = new Foldout {
+                    text = "Advance Options",
+                    style = {
+                        flexDirection = FlexDirection.Column,
+                    },
+                    value = false
+                };
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("invert"), "Invert the UV Tile Discarding."));
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("toggled"), "Only Add Toggled On clip."));
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("dissolveAlpha"), "Use Dissolve UV Dissolve."));
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("renamedMaterial"), "Only required if using renamed properties."));

```

Comment:
Isn't this something we can detect automatically by looking at the material on the renderer? Or maybe not because we're not sure which material is the one they are interested in?

Reply:
Since this is part of the advanced options, it's for people needing more advanced UV tiles toggling since Poiyomi has two systems using UV Tiles. The UV Tile Discard and UV Tile Dissolve (Part of the Dissolve system). Renamed Material is needed if you want to use One Mesh and that one mesh has materials that share the same UV Tile selection.